include/opendht/infohash.h

```diff
@@ -344,22 +344,18 @@ Hash<N>::getRandom(Rd& rdev)
     return h;
 }
 
-struct alignas(std::max_align_t) HexMap : public std::array<std::array<char, 2>, 256> {
-    HexMap() {
-        for (size_t i=0; i<size(); i++) {
-            auto& e = (*this)[i];
-            e[0] = hex_digits[(i >> 4) & 0x0F];
-            e[1] = hex_digits[i & 0x0F];
-        }
-    }
+struct alignas(std::max_align_t) HexMap : public std::array<std::array<char, 2>, 256>
+{
+    static OPENDHT_PUBLIC const HexMap& instance();
```

Comment:
why this change ?

Reply:
As a workaround for the error of linking unexported variable.